src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -334,6 +390,15 @@ Concept
               />
               )
               }
+              <GeneralModel
+                title="Confirm Delete?"
+                content="Are you sure you want to delete"
```

Comment:
Do you think you can include the exact name of what you want to delete?

Reply:
done